### ISSUE TITLE
np.flip requires axis argument

### ISSUE THREAD

**Issue by JuliusKunze** [google/jax#1755]
All of

```python
import numpy as np
np.flip(np.array([1,2]))
np.flip(np.array([1,2]), axis=0)

from jax import numpy as np
np.flip(np.array([1,2]), axis=0)
```

work, but 

```python
from jax import numpy as np
np.flip(np.array([1,2]))
```

doesn't. Is this intended?

**Comment by shoyer** [comment]
This behavior simply hasn't been implemented yet. We would welcome a pull request to add it.

**Comment by shoyer** [comment]
Fixed by #1783
